components/input/client_test.go

```diff
@@ -252,11 +252,11 @@ func TestClient(t *testing.T) {
 
 		_, err = client2.Controls(context.Background(), map[string]interface{}{})
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errControlsFailed.Error())
+		test.That(t, err.Error(), test.ShouldContainSubstring, input.ErrControlsNil(failInputControllerName).Error())
 
 		_, err = client2.Events(context.Background(), map[string]interface{}{})
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errEventsFailed.Error())
+		test.That(t, err.Error(), test.ShouldContainSubstring, input.ErrEventsNil(failInputControllerName).Error())
```

Comment:
Should we still be testing the previous errors or is it not needed?

Reply:
it's not needed because `errControlsFailed` is just an error that was defined in this package and was only returned from the injected input controller. now we are testing if the real function returns the expected error